src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -367,8 +365,6 @@ namespace InternalHeatGains {
                                                       format("Schedule=\"{}\". Minimum is [{:.1R}]. Values must be >= 0.0.", IHGAlphas(3), SchMin));
                                     ErrorsFound = true;
                                 }
-                            }
-                            if (Item1 == 1) {
```

Comment:
[src/EnergyPlus/InternalHeatGains.cc:371]:(style),[duplicateCondition],The if condition is the same as the previous if condition